examples/ConsoleAppWithFailOver/Program.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Azure.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Examples.ConsoleApplicationWithFailOver
+{
+    class Program
+    {
+        static IConfiguration? Configuration { get; set; }
+        private static IConfigurationRefresher? _refresher;
```

Comment:
Not sure why one is property and one is field. Can they both be the field variable?

BTW, why do we need IConfiguration**?**? Isn't IConfiguration nullable already?

Reply:
Not relevant anymore.